### PR TITLE
add header add nav

### PR DESCRIPTION
**Pull request by ric-85**

### CODE REVIEW

**Comment by mz106** [comment]
Hello Ric!

File: src/contact.html [hunk 1]
```diff
@@ -6,5 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+<h1>Contact Us</h1>
+      <nav>
+          <ul>
```
Comment (mz106) [inline src/contact.html:13]: Looks great!

**Review by mz106 — APPROVED** [review]
Approved, good to merge.

File: src/contact.html (hunk 1 above)
Comment (ric-85) [reply src/contact.html:13]: cheers

**Pull request merged**